netkan/netkan/cli.py

```diff
@@ -354,11 +354,16 @@ def auto_freezer(netkan_remote, token, repo, user, days_limit, key):
     help='Collection to put mirrored mods in on Internet Archive',
     required=True,
 )
-def mirrorer(queue, timeout, ckan_meta, ia_access, ia_secret, ia_collection):
+@click.option(
+    '--key', envvar='SSH_KEY', required=True,
+    help='SSH key for accessing repositories',
+)
+def mirrorer(queue, timeout, ckan_meta, ia_access, ia_secret, ia_collection, key):
+    init_ssh(key, Path(Path.home(), '.ssh'))
```

Comment:
Why do we need the SSH key if we don't need to push?

Reply:
We're accessing the repo via SSH, so it needs a key. It's something we should consider though and would be pretty straight forward to do differently with the CLI refactor. No ssh key, provide a https endpoint, SSH key, provide an SSH endpoint.